framework/math/ParallelVector/ParallelSTLVector.cc

```diff
@@ -0,0 +1,479 @@
+#include "ParallelSTLVector.h"
+
+#include "chi_mpi_utils_map_all2all.h"
+#include "data_types/byte_array.h"
+
+#include "chi_log.h"
+#include "chi_log_exceptions.h"
+
+#include <sstream>
+#include <stdexcept>
+#include <cmath>
+#include <numeric>
+
+#define scint64_t static_cast<int64_t>
```

Comment:
If we need to do this, it suggests to me that there are some fundamental design issues here.

Reply:
Its pretty simple actually. DOF-IDs are signed integers, 64bit. It is used 3 times in this file. Once because std::copy needs a "difference"-type. Another two more times to convert unsigned sizes to signed so that dof-ids can be computed.

Reply:
Got it. I actually didn't count the use cases.  3 times is nothing... I was afraid there were numerous uses.  In that case,  it might have been worth it to investigate the target type.